content/en/docs/reference/glossary/dual-stack.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I'm actually a bit dubious about this glossary entry, because it's very Kubernetes flavored.

If I rounded up 10 colleagues and asked them to explain “dual stack” I doubt any of them would mention Pods.
Compare to [Logging](https://kubernetes.io/docs/reference/glossary/?all=true#term-logging), an entry that doesn't use any Kubernetes terminology.

How about leaving this glossary entry out for now?